event_model/tests/test_em.py

```diff
@@ -131,8 +146,14 @@ def test_bulk_events_to_event_page():
     desc_bundle_baseline = run_bundle.compose_descriptor(
         data_keys={'motor': {'shape': [], 'dtype': 'number', 'source': '...'}},
         name='baseline')
+
+    if os.name == 'nt':
+        path_root = 'C:\tmp'
```

Comment:
To avoid it to be evaluated as "tab" (`\t`)
```suggestion
        path_root = 'C:\\tmp'
```

Quick IPython demo:
```py
In [1]: path_root = 'C:\tmp'

In [2]: path_root
Out[2]: 'C:\tmp'

In [3]: print(path_root)
C:	mp

In [4]: path_root = 'C:\\tmp'

In [5]: path_root
Out[5]: 'C:\\tmp'

In [6]: print(path_root)
C:\tmp
```

Reply:
updated